webflows/src/main/java/com/m10s/identity/webflows/tracking/M10sIdentityTrackingEvent.kt

```diff
@@ -0,0 +1,42 @@
+package com.m10s.identity.webflows.tracking
+
+import com.m10s.identity.webflows.BuildConfig
+
+const val PACKAGE_NAME = "identity-sdk-android-web"
+const val PROVIDER_COMPONENT = "m10s-identity"
```

Comment:
```suggestion
const val PROVIDER_COMPONENT = "schibsted-account"
```

for now but might be changed later.